Implementation of Git multiple revisions (#6620)
nextflow-io/nextflow: docs/reference/cli.md

: Service hub where the project is hosted. Options: `gitlab` or `bitbucket`.

`-r` (`master`)

> : Revision to clone - It can be a git branch, tag, or commit SHA number.

Reviewer: ```suggestion
: Revision to clone. Can be a git branch, tag, or commit SHA number.
```

`-properties`
: Print config using Java properties notation.

`-r, -revision`
: :::{versionadded} 25.11.0-edge
  :::

> : Revision of the project (either a git branch, tag or commit SHA number).

Reviewer: ```suggestion
: Project revision. Can be a git branch, tag, or commit SHA number.
```

`-h, -help`
: Print the command usage.

`-r, -revision`
: :::{versionadded} 25.12.0-edge
:::

> : Revision of the project to drop (either a git branch, tag or commit SHA number).

Reviewer: ```suggestion
: Project revision to drop. Can be a git branch, tag, or commit SHA number.
```

  local path  : /Users/evanfloden/.nextflow/assets/nextflow-io/hello
  main script : main.nf
  revisions   :

>   P master (default)

Author: ```suggestion
  * master (default)
```

    mybranch
    testing

>   P v1.1 [t]

Author: ```suggestion
  * v1.1 [t]
```

`-hub` (`github`)
: Service hub where the project is hosted. Options: `gitlab` or `bitbucket`

`-migrate`
:::{versionadded} 25.12.0-edge
  :::
: Update the project asset to new multi-revision strategy.

`-r, -revision`

> : Revision of the project to run (either a git branch, tag or commit SHA number).

Reviewer: ```suggestion
: Project revision to run. Can be a git branch, tag, or commit SHA number.
```

: Enable/disable processes caching.

`-d, -deep`

> : :::{deprecated} 25.11.X-edge

Author: ```suggestion
: :::{deprecated} 25.12.0-edge
  Ignored for new multi-revision asset management strategy. Still used in legacy assets.
```

: Execute the script using the cached results, useful to continue executions that was stopped by an error.

`-r, -revision`

> : Revision of the project to run (either a git branch, tag or commit SHA number).

Reviewer: ```suggestion
: Project revision to run. Can be a git branch, tag, or commit SHA number.
```
